### PR TITLE
Link mistake

### PR DESCRIPTION
**Pull request by chenhonggc**
some link mistake.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6038)

### CODE REVIEW

File: docs/concepts/workloads/controllers/cron-jobs.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ of the set of pods. A cron job does not examine pods at all.
 
 As with all other Kubernetes configs, a cron job needs `apiVersion`, `kind`, and `metadata` fields. For general
 information about working with config files, see [deploying applications](/docs/user-guide/deploying-applications),
-[configuring containers](/docs/user-guide/configuring-containers), and
+[configuring containers](/docs/tasks/configure-pod-container/assign-memory-resource/), and
```
Comment (tengqm) [inline docs/concepts/workloads/controllers/cron-jobs.md]: I don't think the new link is good.

Comment (chenhonggc) [reply docs/concepts/workloads/controllers/cron-jobs.md]: In the docs directory structure：

```shell 
$ ls docs/user-guide/user-guide/configuring*
ls: cannot access docs/user-guide/user-guide/configuring*: No such file or directory
```

Comment (tengqm) [reply docs/concepts/workloads/controllers/cron-jobs.md]: I wasn't saying that the old link should be kept. Either we find a better, meaningful target for the link, or we can delete the link if it is not that relevant any more. Pointing that to memory resource assignment makes no sense to me.

Comment (chenhonggc) [reply docs/concepts/workloads/controllers/cron-jobs.md]: Thanks，restore it.

File: docs/concepts/workloads/controllers/cron-jobs.md [hunk 2]
```diff
@@ -129,7 +129,7 @@ of the set of pods. A cron job does not examine pods at all.
 
 As with all other Kubernetes configs, a cron job needs `apiVersion`, `kind`, and `metadata` fields. For general
 information about working with config files, see [deploying applications](/docs/user-guide/deploying-applications),
-[configuring containers](/docs/tasks/configure-pod-container/assign-memory-resource/), and
+[configuring containers](/docs/user-guide/configuring-containers), and
```
Comment (tengqm) [inline docs/concepts/workloads/controllers/cron-jobs.md]: if the link is pointing to something that doesn't exist, I'd suggest we remove the link.

Comment (chenhonggc) [reply docs/concepts/workloads/controllers/cron-jobs.md]: OK,Thanks!

**Comment by chenhonggc** [comment]
@tengqm master, Thank you for your patient guidance.

**Pull request merged**